cartofacade/testhelpers.go

```diff
@@ -19,14 +19,13 @@ func GetTestConfig(cameraName, movementSensorName string) (CartoConfig, string,
 		ComponentReference: "component",
 		LidarConfig:        TwoD,
 		CloudStoryEnabled:  false,
-		EnableMapping:      false,
+		EnableMapping:      true,
```

Comment:
[note] setting to true so we dont have to provide a apriori map

Reply:
Does this mean that localizing/udpating are no longer being tested?

Reply:
Resolved, looks like it was an artifact of an issue I corrected. Reverting back to original `EnableMapping: false`